tools/_setup_generation/step_refman.py

```diff
@@ -30,6 +30,8 @@ class RefManStep(SetupStep):
 
     # Entries that should be hidden for the time being
     HIDDEN_ENTRIES = ["get_context_id", "invoke_state_callback"]
+    HIDDEN_ENTRIES_FULL = ["taipy.gui.utils._css.get_style"]
```

Comment:
WHat is the difference between `HIDDEN_ENTRIES_FULL` and `HIDDEN_ENTRIES`?

Reply:
Full name vs symbol.
This is a hack to be honest. A case I cannot solve any other way,